docs/exercises/5-multi_region.rst

```diff
@@ -334,10 +337,12 @@ key to illustrate that encryption and decryption is still possible by using a ke
 of the supported resource based access control mechanisms that allow you to programmatically delegate the use of CMKs.
 Grants enable more granular permissions management.
 
-In this portion of the exercise, we will be adding a grant to the local key that checks if the encryption context
-equals {'key use':'bad key'}. Upon local key retrieval, this grant will block this key from being accessed because
-the encryption context supplied with the ciphertext will not match this constraint. Therefore, preventing the local
-key from being accessed.
+In this portion of the exercise, we will be adding a grant to the local key in us-east-2 that will block the key from
```

Comment:
"Local key" nitpick

"we will be adding a grant ... that will block the application's access to use the CMK"?

(I'd kind of like to reword it as "scope down permissions in a way that blocks the application's access", but it's a lot of words.)